static/panes/conformance-view.ts

```diff
@@ -540,20 +536,20 @@ export class Conformance extends Pane<ConformanceViewState> {
 
                             lib.versions = _.pick(lib.versions, (version, versionkey) => {
                                 return versionsInCommon.includes(versionkey);
-                            }) as Record<string, LibraryVersion>;
+                            }) as Record<string, LibraryVersion>; // TODO(jeremy-rifkin)
```

Comment:
(re: TODO comments, this is also cool, attributing to a person. Fun Fact: at Google they track the number of TODOs by username and you get a little badge on your Google internal profile for >500 in the codebase, >1000 etc :)

Reply:
Haha that is awesome